File: Programmers/Lv.2/이진 변환 반복하기/70129.java
class Solution {
    public int[] solution(String s) {
        int[] answer = {};
        
        int zeroCount = 0, count = 0;
        while (!"1".equals(s)) {
            String result = "";
            for (String str : s.split("")) {
                if ("0".equals(str)) {
                    zeroCount++;
                } else {
                    result += str;
                }
            }
            
            count++;
            s = Integer.toBinaryString(result.length());
        }
        
        answer = new int[] {count, zeroCount};
        
        return answer;
    }
}

File: Programmers/Lv.2/이진 변환 반복하기/70129.js
function solution(s) {
    let [count, zeroCount] = [0, 0];
    
    while (s !== '1') {
        const length = s.replace(/0/g, '').length;
        zeroCount += (s.length - length);
        s = length.toString(2);
        count++;
    }
    
    return [count, zeroCount];
}
